taxcalc/growdiff.json

```diff
@@ -0,0 +1,205 @@
+{
+    "_ABOOK": {
+        "long_name": "ABOOK additive difference from default projection",
```

Comment:
Yes, I think additive differences are the best way to express differences in grow factors.  First, this is the way most people talk about differences.  And second, any other way of expressing differences makes it very difficult (and therefore, potentially confusing) to apply **both** baseline and response differences to the grow factors.

@MattHJensen @andersonfrailey